packages/fiori-elements-writer/test/__snapshots__/feop.test.ts.snap

```diff
@@ -0,0 +1,3611 @@
+    <Annotation Term=\\"SAP__common.QuickInfo\\" String=\\"Flight Reference Scenario: Currency Code\\"/>
+</Annotations>
+<Annotations Target=\\"SAP__self.AirlineType/LastChangedAt\\">
+    <Annotation Term=\\"SAP__common.Label\\" String=\\"Time Stamp\\"/>
+    <Annotation Term=\\"SAP__common.QuickInfo\\" String=\\"UTC Time Stamp in Long Form (YYYYMMDDhhmmssmmmuuun)\\"/>
+</Annotations>
+<Annotations Target=\\"SAP__self.AirlineType/LocalCreatedAt\\">
+    <Annotation Term=\\"SAP__common.Label\\" String=\\"Time Stamp\\"/>
+    <Annotation Term=\\"SAP__common.QuickInfo\\" String=\\"UTC Time Stamp in Long Form (YYYYMMDDhhmmssmmmuuun)\\"/>
+</Annotations>
+<Annotations Target=\\"SAP__self.AirlineType/LocalLastChangedAt\\">
+    <Annotation Term=\\"SAP__common.Label\\" String=\\"Time Stamp\\"/>
+    <Annotation Term=\\"SAP__common.QuickInfo\\" String=\\"UTC Time Stamp in Long Form (YYYYMMDDhhmmssmmmuuun)\\"/>
+</Annotations>
+<Annotations Target=\\"SAP__self.CurrencyType/Decimals\\">
+    <Annotation Term=\\"SAP__common.Label\\" String=\\"Decimal Places\\"/>
+    <Annotation Term=\\"SAP__common.QuickInfo\\" String=\\"Number of decimal places\\"/>
+</Annotations>
+</Schema>
+</edmx:DataServices>
+</edmx:Edmx>",
+    "state": "modified",
+  },
+  "webapp/manifest.json": Object {
+    "contents": "{
+  \\"_version\\": \\"1.37.0\\",
+  \\"sap.app\\": {
+    \\"id\\": \\"fefeop1\\",
+    \\"type\\": \\"application\\",
+    \\"i18n\\": \\"i18n/i18n.properties\\",
+    \\"applicationVersion\\": {
+      \\"version\\": \\"0.0.1\\"
+    },
+    \\"title\\": \\"{{appTitle}}\\",
+    \\"description\\": \\"{{appDescription}}\\",
+    \\"resources\\": \\"resources.json\\",
+    \\"sourceTemplate\\": {
+      \\"id\\": \\"test-fe-template\\",
+      \\"version\\": \\"1.2.3-test\\"
+    },
+    \\"dataSources\\": {
+      \\"mainService\\": {
+        \\"uri\\": \\"sap/opu/odata4/dmo/sb_travel_mduu_o4/srvd/dmo/sd_travel_mduu/0001/\\",
+        \\"type\\": \\"OData\\",
+        \\"settings\\": {
+          \\"annotations\\": [
+            \\"annotation\\"
+          ],
+          \\"localUri\\": \\"localService/metadata.xml\\",
+          \\"odataVersion\\": \\"4.0\\"
+        }
+      },
+      \\"annotation\\": {
+        \\"type\\": \\"ODataAnnotation\\",
+        \\"uri\\": \\"annotations/annotation.xml\\",
+        \\"settings\\": {
+          \\"localUri\\": \\"annotations/annotation.xml\\"
+        }
+      }
+    }
+  },
+  \\"sap.ui\\": {
+    \\"technology\\": \\"UI5\\",
+    \\"icons\\": {
+      \\"icon\\": \\"\\",
+      \\"favIcon\\": \\"\\",
+      \\"phone\\": \\"\\",
+      \\"phone@2\\": \\"\\",
+      \\"tablet\\": \\"\\",
+      \\"tablet@2\\": \\"\\"
+    },
+    \\"deviceTypes\\": {
+      \\"desktop\\": true,
+      \\"tablet\\": true,
+      \\"phone\\": true
+    }
+  },
+  \\"sap.ui5\\": {
+    \\"flexEnabled\\": true,
+    \\"dependencies\\": {
+      \\"minUI5Version\\": \\"1.90.0\\",
```

Comment:
you say 1.90 here but in the ui5 yaml you point to 1.86, you may want to choose one :)